python_builtin_functions.py

```diff
@@ -4,86 +4,170 @@ def myabs(number):
         return -number
     return number
 
+
 def myall(iterable):
     """ Implementation of built-in `all` function """
-    pass
+    for item in iterable:
+        if not item:
+            return False
+    return True
 
 
 def myany(iterable):
     """ Implementation of built-in `any` function """
-    pass
+    for item in iterable:
+        if item:
+            return True
+    return False
 
 
 def mylen(s):
     """ Implementation of built-in `len` function """
-    pass
+    i=0
+    for _ in s:
+        i+=1
+    return i
 
 
 def mysum(iterable):
     """ Implementation of built-in `sum` function """
-    pass
+    i=0
+    for x in iterable:
+        i+=x
+    return i
 
 
 def mymax(iterable):
     """ Implementation of built-in `max` function """
-    pass
+    i=0
```

Comment:
This initialisation of `i` to zero doesn't work for a list containing only negative integers..

Instead, initialise `i` to the first element , thereby assuming the max element as first element and comparing the max element with rest of the list items.

Reply:
Changed as per your instruction.